docs/modules/deploy/pages/deploying-on-aws.adoc

```diff
@@ -158,6 +158,9 @@ clientConfig.getNetworkConfig().getAwsConfig()
 --
 ====
 
+Hazelcast Client on EC2 can discover the members running in ECS cluster. For doing so you need specify the needed <<ecs_properties,ECS related properties>> (`cluster`, `family`, `service-name`) and make sure to <<ecs_permissions,required permissions>>.
+If none of the ECS or EC2 related properties are specified, the AWS discovery will try to discover the EC2 memebr first, and then, if no EC2 members found, it will try to discover ECS members.
```

Comment:
If none of the ECS or EC2 related properties are specified, the AWS discovery tries to discover the EC2 members. If none is found, then it tries to discover ECS members.